ls1intum/Artemis
# fixes hint placement for multiple choice questions

### Checklist


### Motivation and Context


Fixes the issue with the multiple choice quiz hint placement. Fixes issues 1 and 4 of [the list of found exam mode bugs.](https://confluence.ase.in.tum.de/pages/viewpage.action?spaceKey=ArTEMiS&title=Issues+Student+Subteam)


### Description

Fixes the placement of the quiz hints.

### Steps for Testing


1. Create an exam with a multiple choice questions with hints.
2. participate with a student
3. you should always be able to see both types of hints fully, like shown in the screenshots below.

**Instead of creating an exam you can also just create a quiz with a multiple choice question. Then reduce the window size and check if the all hints are still displayed correctly.**

![image](https://user-images.githubusercontent.com/33342534/125067760-f7240480-e0b4-11eb-9708-53f965f4df54.png)
![image](https://user-images.githubusercontent.com/33342534/125067812-04d98a00-e0b5-11eb-9d30-a528a4ba70f9.png)
